### PR TITLE
Additional rewards

### PR DESCRIPTION
**Pull request by WT-MM**
Action smoothing
Torque penalty
Fixed height reward

### CODE REVIEW

File: ksim/rewards.py [hunk 1]
```diff
@@ -115,8 +118,42 @@ class BaseHeightReward(Reward):
...
+        current_actions = trajectory.action
+
+        # Shift actions to get previous actions (pad with first action)
+        previous_actions = jnp.concatenate(
+            [
+                current_actions[..., :1, :],  # First action
+                current_actions[..., :-1, :],  # Previous actions for remaining timesteps
+            ],
+            axis=-2,
+        )
```
Comment (codekansas) [inline ksim/rewards.py:145]: can also consider something like `jnp.roll`

File: ksim/task/rl.py [hunk 2]
```diff
@@ -590,6 +590,7 @@ def step_engine(
         trajectory = Trajectory(
             qpos=next_physics_state.data.qpos,
             qvel=next_physics_state.data.qvel,
+            actuator_frc=next_physics_state.data.actuator_force,
```
Comment (codekansas) [inline ksim/task/rl.py]: oh... maybe we should figure out a better way to populate the trajectory during runtime... will noodle on this

edit: maybe the best way to do this is to check `obs` for a particular key - the reward here should throw an error if the relevant key does not exist. so, we should add some `obs` which gets `actuator_force`

**Review by codekansas — APPROVED** [review]

File: ksim/task/rl.py (hunk 2 above)
Comment (budzianowski) [reply ksim/task/rl.py]: What about information that is not observation available at run time? I guess we can always add it to the critic?
